apps/prairielearn/src/pages/partials/instructorInfoPanel.ejs

```diff
@@ -67,6 +68,7 @@
     <div class="pr-1">Title:</div>
     <div><%= question.title %></div>
   </div>
+  <% if (typeof variant !== "undefined") { %>
```

Comment:
This will hide information associated with the variant if there is no variant; I don't think we can do anything better?

Reply:
This makes sense.